bukuserver/forms.py

```diff
@@ -47,54 +81,58 @@ class SwapForm(FlaskForm):
     id2 = HiddenField(filters=[int])
 
 
-class ApiTagForm(FlaskForm):
-    class Meta:
-        csrf = False
-
-    tags = FieldList(StringField(validators=[DataRequired(), validate_tag]), min_entries=1)
-
-    tags_str = None
+class ApiFetchDataForm(Form):
+    url = StringField(validators=[DataRequired()])
```

Comment:
I actually considered using `UrlField` here, but it imposed additional requirements on the value that isn't present elsewhere in the app